app/javascript/packs/banner_editor.jsx

```diff
@@ -181,11 +188,12 @@ export default class BannerEditor extends React.Component {
 
     let that = this
 
-    let url = '/publishers/' + document.getElementById("publisher_id").value + "/site_banners/fetch";
+    let url = '/publishers/' + document.getElementById("publisher_id").value + "/site_banners/fetch?channel_id=" + this.props.channels[this.state.channelIndex].id;
```

Comment:
Can we move the publisher_id to a prop that gets passed in?

Reply:
Done ✔️